docs/apps/lnw/es2k/es2k-lnw-underlay-lag.md

```diff
@@ -0,0 +1,378 @@
+<!--
+
+ Copyright 2024 Intel Corporation.
+
+ SPDX-License-Identifier: Apache-2.0
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at:
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+/
+- -->
+
+# Linux Networking with LAG (ES2K)
+
+This document explains how to run the Linux networking scenario with LAG (Link Aggregation Group) on ES2K.
+
+## Topology
+
+![Linux Networking Topology](es2k-lnw-topology.png)
+
+See [Linux Networking for ES2K](es2k-linux-networking.md)
+for more details on this feature.
+
+Prerequisites:
+
+- Download `hw-p4-programs` TAR file specific to the build and extract it to get `fxp-net_linux-networking-v2` p4 artifacts. Go through `Limitations` specified in `README` and bring up the setup accordingly.
+- Follow steps mentioned in [Deploying P4 Programs for E2100](/guides/es2k/deploying-p4-programs) for bringing up IPU with a custom P4 package.
```

Comment:
deploying -p4-programs link is throwing 404 error

Reply:
From the docs README file:

The files in this directory are used to build the [P4 Control Plane User Guide](https://ipdk.io/p4cp-userguide/).

You can view the source files individually, but the internal links are relative to the root of the user guide. Many of them won't work correctly outside the context of the finished document.

We recommend that you access the User Guide through the link above.